WebApp/autoreduce_webapp/reduction_viewer/views.py

```diff
@@ -70,7 +71,11 @@ def index(request):
             return_url = default_next
     elif request.GET.get('sessionid'):
         request.session['sessionid'] = request.GET.get('sessionid')
-        user = authenticate(token=request.GET.get('sessionid'))
+        try:
+            user = authenticate(token=request.GET.get('sessionid'))
+        except ICATConnectionException as excep:
+            return render_error(request, str(excep))
```

Comment:
Can we log this

Reply:
I think it will be logged already here https://github.com/ISISScientificComputing/autoreduce/blob/a8d26282cf6a3a2b2354166da62434b8d4087557/WebApp/autoreduce_webapp/autoreduce_webapp/icat_cache.py#L58